charts/customer-center/README.md

```diff
@@ -0,0 +1,126 @@
+
+Chart for Customer-Center application
+
+**Homepage:** <https://github.com/adfinis-sygroup/customer-center>
+
+## Maintainers
+This chart is maintained by [Adfinis](https://adfinis.com/?pk_campaign=github&pk_kwd=helm-charts).
+
+## Source Code
+
+* <https://github.com/adfinis-sygroup/helm-charts/tree/master/charts/customer-center>
+
+## Requirements
+
+| Repository | Name | Version |
+|------------|------|---------|
+| https://charts.bitnami.com/bitnami | postgresql | ~10.3.11 |
+| https://charts.bitnami.com/bitnami | redis | ~12.8.3 |
+
+## Values
+
+| Key | Type | Default | Description |
+|-----|------|---------|-------------|
+| backend.image.pullPolicy | string | `"IfNotPresent"` | Backend image pull policy |
+| backend.image.repository | string | `"ghcr.io/adfinis-sygroup/customer-center/backend"` | Backend image name |
+| backend.livenessProbe.enabled | bool | `true` | Enable liveness probe on backend |
+| backend.livenessProbe.failureThreshold | int | `6` | Number of tries to perform the probe |
+| backend.livenessProbe.initialDelaySeconds | int | `60` | Number of seconds after the container has started before liveness probe is initiated |
+| backend.livenessProbe.path | string | `"/api/v1/health"` | Path to probe |
+| backend.livenessProbe.periodSeconds | int | `10` | How often (in seconds) to perform the probe |
+| backend.livenessProbe.successThreshold | int | `1` | Minimum consecutive successes for the probe to be considered successful after having failed |
+| backend.livenessProbe.timeoutSeconds | int | `5` | Number of seconds after which the probe times out |
+| backend.readinessProbe.enabled | bool | `false` | Enable readiness probe on backend |
+| backend.readinessProbe.failureThreshold | int | `6` | Number of tries to perform the probe |
+| backend.readinessProbe.initialDelaySeconds | int | `30` | Number of seconds after the container has started before readiness probe is initiated |
+| backend.readinessProbe.path | string | `"/api/v1/health"` | Path to probe |
+| backend.readinessProbe.periodSeconds | int | `10` | How often (in seconds) to perform the probe |
+| backend.readinessProbe.successThreshold | int | `1` | Minimum consecutive successes for the probe to be considered successful after having failed |
+| backend.readinessProbe.timeoutSeconds | int | `5` | Number of seconds after which the probe times out |
+| backend.replicaCount | int | `1` | (int) Number of Backend replicas |
+| backend.resources | object | `{}` | Resource limits for backend |
+| backend.service.externalPort | int | `3000` | External Port of backend service |
+| backend.service.internalPort | int | `3000` | Internal Port of backend service |
+| backend.service.type | string | `"ClusterIP"` |  |
```

Comment:
why was the description removed?

Reply:
https://github.com/adfinis-sygroup/helm-charts/pull/188#discussion_r593317461

Reply:
This has been fixed and added back.